### ISSUE TITLE
Problems when uninstall

### ISSUE THREAD

**Issue by akBaroks** [files-community/Files#11984]
### Description

I uninstall the bar because a massives frezze, and when i apply the register bach for remove file from my integration, the file has been corrupted because a 2 letters in the reg file because @=- is not in the reg file, i'm added am problm solved

### Steps To Reproduce

Remove integration with windows

### Requirements

; Revert Default (old)
[HKEY_CURRENT_USER\SOFTWARE\Classes\Directory\shell]
`@=-`

[-HKEY_CURRENT_USER\SOFTWARE\Classes\Directory\shell\openinfiles]
`@=-`
; Revert Win+E
[-HKEY_CURRENT_USER\SOFTWARE\Classes\CLSID\{52205fd8-5dfb-447d-801a-d0b52f2e83e1}]
`@=-`

### Files Version

2.4.60.0

### Windows Version

Windows 10

### Log file

NO LOG

**Comment by yaira2** [comment]
Did you turn off the setting before you uninstalled? It's supposed to auto revert but the recommendation is to manually turn it off.

**Comment by akBaroks** [comment]
It was not possible through the /Application button, I had to do it from the registry.
When I clicked the button, it stayed on again, and never exited, as if it were bugged, while all this was happening, I was getting the error "shell:::[52205fd8-447d-801a-d0b52f22e83e1]" next to "Explorer .exe does not have a default application to open it."
Everything was solved, when opening the registry file to uninstall, check and see that the last 3 keys do not return the default parameter.

**Comment by Josh65-2201** [comment]
Admin is needed for the changes to do anything. Normal it shows the UAC prompt. Did you disable that too?

**Comment by akBaroks** [comment]
It did show it but it was like it wouldn't work.
In the application, the first time I press it to integrate, and it shows the UAC and the button is activated and deactivated.
So I activated it by registry, but when I deactivated it I had the same problem so I deactivated it from the registry file, and when doing so the file was corrupted or badly armed and by not restoring the 3 final values ​​it did not let explorer.exe work

**Comment by Josh65-2201** [comment]
Have to tried reinstalling Files

**Comment by akBaroks** [comment]
Yes, and it could not be integrated again, in fact I spent 8 hours looking for the problem since I had run out of "explorer", I couldn't directly access anything.
Until I look at the app folder, and found these 2 files.

**Comment by yaira2** [comment]
@akBaroks what happens when you try enabling the setting again through the settings page?

**Comment by yaira2** [comment]
@akBaroks

**Comment by akBaroks** [comment]
After uninstall?
I reinstall because i don't have chance to execute any file, and never the button work, when i execute the button, the program show UAC, then the button putos, on and rapidly puts off. nothing work.

Nothing,

**Comment by yaira2** [comment]
Can you share attach a screen recording? I'm having a hard time reproducing this issue so it might help to see a screen recording.
